client/src/Pages/MyPage/index.tsx

```diff
@@ -0,0 +1,172 @@
+import { login } from '~/lib/api/auth';
+import {
+  UsersGetResponseBody,
+  UsersPutResponseBody,
+} from '~/lib/api/types/users';
+import {
+  REG_EMAIL,
+  WARNING_EMAIL,
+  REG_IMAGE,
+  REG_PHONE,
+} from '~/utils/validation';
+import { alert } from '~/utils/modal';
+import UserInfoInput from '~/components/UserInfoInput';
+import PhoneInput from '~/components/PhoneInput';
+import EmailInput from '~/components/EmailInput';
+import {
+  StyledMyPage,
+  Title,
+  RowWrapper,
+  RowTitle,
+  PhotoWrapper,
+  ImagePreview,
+  ImageDesc,
+  ImageInput,
+} from './index.style';
+
+const message = {
+  EMAIL_UPDATE_SUCCESS: '이메일이 수정되었습니다.',
+  PHONE_UPDATE_SUCCESS: '연락처가 수정되었습니다.',
+  PHOTO_UPDATE_SUCCESS: '사진이 수정되었습니다.',
+  NOT_CORRECT_IMAGE: '올바른 이미지파일 형식이 아닙니다.',
+  EMAIL_UPDATE_FAIL: '이메일 수정에 실패했습니다.',
+  PHONE_UPDATE_FAIL: '연락처 수정에 실패했습니다.',
+};
+
+const MyPage: React.FC = () => {
+  const [profile, setProfile] = useState<string>(
+    'https://user-images.githubusercontent.com/47776356/129712816-13701b24-57cc-451e-93c6-ca7afe190af1.jpeg',
+  );
+
+  const [userInfo, setUserInfo] = useState<UsersGetResponseBody | null>(null);
+  const imagePreviewRef: RefObject<HTMLLabelElement> = useRef();
+
+  const handleSubmitEmail = async (value: string) => {
+    const response = await putMe({ email: value });
+    if (response.statusCode === 200) {
+      setUserInfo({ ...userInfo, email: value });
+      alert(message.EMAIL_UPDATE_SUCCESS);
+    }
+  };
+
+  const handleSubmitPhone = async (value: string) => {
+    const response = await putMe({ phone: value });
+    if (response.statusCode === 200) {
+      alert(message.PHONE_UPDATE_SUCCESS);
+      setUserInfo({ ...userInfo, phone: value });
+    }
+  };
+
+  const handleSubmitProfile = async (file, fileToString) => {
+    const response = await putMe({
+      img: file,
+    });
+
+    if (response.statusCode === 200) {
+      setProfile(fileToString);
+      alert(message.PHOTO_UPDATE_SUCCESS);
+    }
+  };
+
+  const handleImageInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files[0];
+    if (!file.type.match(REG_IMAGE)) {
+      e.target.value = '';
+      alert(message.NOT_CORRECT_IMAGE);
+      return;
+    }
+    const reader = new FileReader();
+    reader.readAsDataURL(e.target.files[0]);
+    reader.onload = (event) => {
+      handleSubmitProfile(file, event.target.result.toString());
+    };
+  };
+
+  const emailValidator = (value: string) => {
+    return REG_EMAIL.test(value);
+  };
+  const phoneValidator = (value: string) => {
+    return REG_PHONE.test(value);
+  };
+
+  useEffect(() => {
+    imagePreviewRef.current.style.backgroundImage = `url(${profile})`;
+  }, [imagePreviewRef, profile]);
+
+  useEffect(() => {
+    const fetchUser = async () => {
+      // 로그인 기능이 아직 구현되어있지 않아, 테스트를 위해 로그인 로직을 추가해놓았습니다.
+      // const test = await login({
+      //   id: 'test',
+      //   password: 'hahahahoho',
+      // });
```

Comment:
이런 경우는 TODO: (@TODO) 나 FIXME: (@FIXME) 를 붙여주면 좀더 주석의 의미를 명확히 전달할 수 있습니다 :)

```suggestion
      // TODO: 로그인 기능이 아직 구현되어있지 않아, 테스트를 위해 로그인 로직을 추가해놓았습니다.
      // const test = await login({
      //   id: 'test',
      //   password: 'hahahahoho',
      // });
```
```suggestion
      // @TODO 
      // 로그인 기능이 아직 구현되어있지 않아, 테스트를 위해 로그인 로직을 추가해놓았습니다.
      // const test = await login({
      //   id: 'test',
      //   password: 'hahahahoho',
      // });
```

Reply:
그렇군요. 반영하겠습니다.